src/routes/EResourceViewRoute.js

```diff
@@ -159,11 +167,14 @@ class EResourceViewRoute extends React.Component {
   }
 
   getRecords = (resource) => {
-    return get(this.props.resources, `${resource}.isPending`, true)
-      ?
-      undefined
-      :
-      get(this.props.resources, `${resource}.records`);
+    const records = get(this.props.resources, `${resource}.records`, []);
```

Comment:
Doesn't this now change the behaviour of showing the loader for entitlements and entitlement options? It looks like we'll show the first page while we're fetching the full list. This makes me lean more [towards my second idea here](https://github.com/folio-org/ui-agreements/pull/404/files#diff-06b0e501c8006ad08ddf6c1656ad19acR206) since we could revert these changes.